### PR TITLE
Added Alpha K Slack Extension

### PR DESCRIPTION
**Pull request by vithu30**

### CODE REVIEW

File: component/src/main/java/org/wso2/extension/siddhi/execution/reorder/utils/WindowCoverage.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
+package org.wso2.extension.siddhi.execution.reorder.utils;
```
Comment (harsha89) [inline component/src/main/java/org/wso2/extension/siddhi/execution/reorder/utils/WindowCoverage.java:1]: Do we required to have the license?

File: component/src/main/java/org/wso2/extension/siddhi/execution/reorder/utils/WindowCoverage.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    }
+
+    /**
+     * Calculate Window Coverage
+     *
+     * @param eventTimestamps
+     * @param windowSize
+     * @return runtimeWindowCoverage
+     */
+
```
Comment (harsha89) [inline component/src/main/java/org/wso2/extension/siddhi/execution/reorder/utils/WindowCoverage.java:93]: no need of new line

File: component/src/main/java/org/wso2/extension/siddhi/execution/reorder/utils/WindowCoverage.java (hunk 1 above)
Comment (vithu30) [reply component/src/main/java/org/wso2/extension/siddhi/execution/reorder/utils/WindowCoverage.java:1]: We need to have license header. Fixed in #38

File: component/src/main/java/org/wso2/extension/siddhi/execution/reorder/utils/WindowCoverage.java (hunk 2 above)
Comment (vithu30) [reply component/src/main/java/org/wso2/extension/siddhi/execution/reorder/utils/WindowCoverage.java:93]: Fixed in #38

**Pull request merged**